plugins/module_utils/openapi.py

```diff
@@ -42,9 +42,15 @@ def __init__(
         validate_certs=True,
         refresh_cache=False,
     ):
-        self.base_url = base_url
         self.doc_path = doc_path
 
+        if base_url.startswith("unix:"):
+            self.unix_socket = base_url.replace("unix:", "")
+            self.base_url = "http://localhost/"
```

Comment:
Interesting that this is needed on unix-sockets.

Reply:
If you are interested https://github.com/ansible/ansible/pull/43560 is the PR that introduce this behavior and made it possible

Reply:
OK, I'll take your word for it. Seems like your folks will run kind of a nightly test for this. ;)